tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -1191,6 +1191,345 @@ def test_OnDiskDataset_preprocess_homogeneous(edge_fmt):
         fused_csc_sampling_graph = None
 
 
+@pytest.mark.parametrize("edge_fmt", ["csv", "numpy"])
```

Comment:
no need to test on `edge_fmt` as this is not the target we add the hardcode testcase.